src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx

```diff
@@ -423,17 +424,31 @@ describe('526 helpers', () => {
     });
   });
 
-  describe('isUploadingPtsdForm', () => {
-    it('should return true if user has chosen to upload documents', () => {
+  describe('isUploading781Form', () => {
+    it('should return true if user has chosen to upload 781', () => {
       const formData = {
-        'view:uploadPtsdChoice': 'upload',
+        'view:upload781Choice': 'upload',
       };
-      expect(isUploadingPtsdForm(formData)).to.be.true;
+      expect(isUploading781Form(formData)).to.be.true;
     });
 
-    it('should return false if user has not chosen to upload documents', () => {
+    it('should return false if user has not chosen to upload 781', () => {
       const formData = {};
-      expect(needsToEnter781({ formData })).to.be.false;
+      expect(isUploading781Form({ formData })).to.be.false;
```

Comment:
I hate to be a pain--and I know this wasn't introduced in this PR--but this shouldn't be destructured. :sweat_smile: